src/hooks/useValidateAppVersion.ts

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import { Linking, Platform } from 'react-native';
+import Config from 'react-native-config';
+import DeviceInfo from 'react-native-device-info';
+
+import remoteConfig from '@react-native-firebase/remote-config';
+
+enum RemoteFields {
+  MINIMUM_VERSION = '',
+}
+
+export const useValidateAppVersion = () => {
+  const [isUpdateAvailable, setIsUpdateAvailable] = useState<boolean | undefined>(false);
+
+  const openStore = () => {
+    Platform.select({
+      ios: Linking.openURL(Config.APP_STORE_URL),
+      android: Linking.openURL(Config.PLAY_STORE_URL),
+    });
+  };
+
+  const checkUpdate = async () => {
+    try {
+      remoteConfig().setDefaults({
+        minimum_version: '',
+      });
+
+      await remoteConfig().fetchAndActivate();
+
+      const remoteVersion = remoteConfig().getValue(RemoteFields.MINIMUM_VERSION).asString();
+      const localVersion = DeviceInfo.getVersion();
+      const updateAvailable = compareVersions(localVersion, remoteVersion);
+      setIsUpdateAvailable(updateAvailable);
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  function compareVersions(local: string, remote: string) {
```

Comment:
You can convert this function to an arrow function, as you did with the other two functions.
That's to keep consistency with the function definition.

Reply:
Done